package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "cfpb-design-system",
-  "private": "true",
+  "version": "1.0.0",
```

Comment:
I set this to `1.0.0` as that is the same version set in `docs/package.json`.

Reply:
Hmm... shouldn't it be `0.0.8` to match the versions of the individual component packages?

Reply:
@Scotchester that was true when the project root was also the master `capital-framework` component but we now have the master component in a [sub dir](https://github.com/cfpb/design-system/tree/master/packages/cfpb-design-system) so the version number here doesn't matter.